cmd/server/search_handlers.go

```diff
@@ -338,16 +341,38 @@ var (
 		},
 	}
 
-	// eu - consolidated sanctions list
+	// uk - consolidated sanctions list
 	ukGatherings = []searchGather{
 		func(s *searcher, _ filterRequest, limit int, minMatch float64, name string, resp *searchResponse) {
 			resp.UKCSL = s.TopUKCSL(limit, minMatch, name)
 		},
+		func(s *searcher, _ filterRequest, limit int, minMatch float64, name string, resp *searchResponse) {
+			resp.UKSanctionsList = s.TopUKSanctionsList(limit, minMatch, name)
+		},
 	}
 
-	allGatherings = append(baseGatherings, euGatherings...)
+	allGatherings = concateMultipleSlices([][]searchGather{baseGatherings, cslGatherings, euGatherings, ukGatherings})
 )
 
+// this function taken from here https://freshman.tech/snippets/go/concatenate-slices/
+// and modified to be specific to searchGather slices
+func concateMultipleSlices(slices [][]searchGather) []searchGather {
+	var totalLen int
+	for _, s := range slices {
+		totalLen += len(s)
+	}
+
+	result := make([]searchGather, totalLen)
+
+	var i int
+
+	for _, s := range slices {
+		i += copy(result[i:], s)
+	}
+
+	return result
+}
```

Comment:
I'm confused why this is needed. We should be able to append without a helper func. `append(arr, items...)`

Reply:
This helper appends multiple slices together. In the case of the searchGathers when a new list is added and a searchGather for it, there will need to be more variables to add each of them. For instance,
```
baseGather = append(cslGatherers, someOtherGatherers...)
euGather = []searchGather
ukGather = []searchGather
allGather = append(baseGather, euGather...)
allAllGather = append(allGather, ukGather...)
```
the intermediate lists take up space and aren't as intuitive as just having a helper functions that allocates a list of the total size of the sub lists and appends them all.